sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -103,7 +103,7 @@ def __init__(self, name, init_config, instances):
         self.reported_hostname = self.instance.get('reported_hostname')
         self.autodiscovery = is_affirmative(self.instance.get('database_autodiscovery'))
         self.autodiscovery_include = self.instance.get('autodiscovery_include', ['.*'])
-        self.autodiscovery_exclude = self.instance.get('autodiscovery_exclude', [])
+        self.autodiscovery_exclude = self.instance.get('autodiscovery_exclude', ['model'])
```

Comment:
i think some of the autodiscovery tests rely on this db unfortunately :(

Reply:
`:homer-screaming:`